engine/apps/telegram/alert_group_representative.py

```diff
@@ -41,7 +41,6 @@ def get_handlers_map():
             AlertGroupLogRecord.TYPE_AUTO_UN_ACK: "alert_group_action",
             AlertGroupLogRecord.TYPE_RESOLVED: "alert_group_action",
             AlertGroupLogRecord.TYPE_UN_RESOLVED: "alert_group_action",
-            AlertGroupLogRecord.TYPE_ACK_REMINDER_TRIGGERED: "alert_group_action",
```

Comment:
We don't have reminders for telegram, do we? (so was this doing nothing?)

Reply:
Correct, it's a slack-specific feature